test/conversations_page_test.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:adventures_in_chat_app/conversations_page.dart';
+import 'package:adventures_in_chat_app/models/conversation_item.dart';
+import 'package:adventures_in_chat_app/services/database_service.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+
+import 'chat_page_test.dart';
+import 'mocks/database_mocks.dart';
+import 'utils/image_test_utils.dart';
+
+void main() {
+  group('ConversationsList', () {
+    testWidgets('find conversationId, uids, displaynames and photoURLs',
```

Comment:
The test description doesn't describe the test in a way I understand what it's doing - we don't have any documentation for the tests so I think it's important we have descriptions that describe the tests clearly so you can get what is being tested from the description.

Reply:
I imagine you might be getting annoyed by these requests - I'm not meaning to be difficult I really do think this is worth bothering about as the tests are often the best way to figure out how parts of the code work and having a description that you can glance at and know what the test is doing makes the process of finding the relevant test so much easier. Please feel free to ask me to do anything, I'm not meaning to create an annoying burden.